closure/goog/net/jsloader.js

```diff
@@ -94,36 +94,26 @@ goog.net.jsloader.scriptsToLoad_ = [];
  * @param {Array<string>} uris The URIs to load.
  * @param {goog.net.jsloader.Options=} opt_options Optional parameters. See
  *     goog.net.jsloader.options documentation for details.
+ * @return {!goog.async.Deferred} The deferred result, that may be used to add
+ *     callbacks
  */
 goog.net.jsloader.loadMany = function(uris, opt_options) {
   // Loading the scripts in serial introduces asynchronosity into the flow.
-  // Therefore, there are race conditions where client A can kick off the load
-  // sequence for client B, even though client A's scripts haven't all been
-  // loaded yet.
-  //
-  // To work around this issue, all module loads share a queue.
+  // The client receives a deferred to manage parallel loads.
   if (!uris.length) {
-    return;
-  }
-
-  var isAnotherModuleLoading = goog.net.jsloader.scriptsToLoad_.length;
```

Comment:
Ok. I had asked for feedback on the google group. It's a very specific use-case. The problem I see is it blocks other use-cases, as the caller has no control. Do you agree w/ some option to use/skip the queue? A param (opt_queue), or maybe an add'l func? Tx



> On 11.08.2015, at 18:58, Joel Martinez <notifications@github.com> wrote:
> 
> In closure/goog/net/jsloader.js:
> 
> >    if (!uris.length) {
> > -    return;
> > -  }
> > -
> > -  var isAnotherModuleLoading = goog.net.jsloader.scriptsToLoad_.length;
> Ehhh, I think I'd prefer you not change the behavior of this method this radically. The original impl was designed with a specific use case in mind and breaking that assumption could cause breakages internally.
> 
> Can you revert this back to the original fix w/ only @return added?
> 
> —
> Reply to this email directly or view it on GitHub.
>